### DISCUSSION TITLE
Align easepick on the right end of an element?

### DISCUSSION THREAD

**Discussion by hajo-travanto** [easepick/easepick#83] [Q&A]
Hi, by default easepick is aligning to the left start of an element it is bound to. Is there a way to make this the right side? So that the picker will use more width on the left side of the website and stick to the right end of an element?

**Comment by wakirin** [comment]
Currently (v1.2.0) there is no such option

**Comment by hajo-travanto** [comment]
Would you say that this could be something for a feature request or it should be solved outside of easepick?
